Test.h

```diff
@@ -0,0 +1,10 @@
+#ifndef TEST_H_
+#define TEST_H_
+
+#include "color.h"
+
+extern void testNumberToPair(int pairNumber,MajorColor_tst expectedMajor, MinorColor_tst expectedMinor);
```

Comment:
Try out test names to include the expectation. See if more test possibilities emerge.